utils/date.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-confusing-arrow */
+
+const formatDate = (inputDateString) => {
```

Comment:
Can you change this to the other `formatDate` that's now in `master`? I guess you would need to get an i18n-aware Node.js first, though...